multi/imxrt-multi/cm4.c

```diff
@@ -107,10 +110,11 @@ static inline int fifo_popPacket(packet_t *packet, fifo_t *fifo)
 }
 
 
-static inline int fifo_push(char byte, fifo_t *fifo)
+static inline int fifo_push(unsigned char byte, fifo_t *fifo)
 {
-	if ((fifo->wptr + 1) % sizeof(fifo->buff) == fifo->wptr)
+	if (((fifo->wptr + 1) % sizeof(fifo->buff)) == fifo->wptr) {
```

Comment:
Doing `(fifo->wptr + 1) % sizeof(fifo->buff)` twice is not effective, maybe use temporary variable to store the `nextwptr`.

Reply:
Fixed, as a side note, there was a nasty bug hiding in a plain sight, fixed ;)

Reply:
That's right.